eox_tenant/settings/aws.py

```diff
@@ -61,6 +59,9 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
         settings.SITE_ID
     )
 
+    if DJANGO_CURRENT_SITE_MIDDLEWARE in settings.MIDDLEWARE_CLASSES:
```

Comment:
Do you think we should be doing this in common.py instead of aws.py?

Reply:
to keep the current pattern IMO we should do this in aws, if we want to move this to common we should move all the middlewares and this is out of the scope of this pr

Reply:
I think that this have to be enabled or disabled with a setting. Maybe it could be the setting _USE_EOX_TENANT_